compiler/src/dotty/tools/dotc/printing/RefinedPrinter.scala

```diff
@@ -815,14 +815,16 @@ class RefinedPrinter(_ctx: Context) extends PlainPrinter(_ctx) {
           if isExtension then
             val paramss =
               if tree.name.isRightAssocOperatorName then
-                // we have the encoding: leadingUsingOrTypeParams rightParam trailingUsing leftParam
-                // we need to swap rightParam and leftParam
-                val (leadingUsing, rest1) = tree.paramss.span(isUsingOrTypeParamClause)
-                val (rightParamss, rest2) = rest1.splitAt(1)
-                val (trailingUsing, rest3) = rest2.span(isUsingClause)
-                val (leftParamss, rest4) = rest3.splitAt(1)
+                // we have the encoding: leadingTy leadingUsing rightTyParamss rightParamss leftParamss trailingUsing
```

Comment:
What is `leadingTy`? Typo?

Reply:
Would also be good to give an example here in a comment.

Reply:
I've added comments and used better names